### PR TITLE
Added download button to download image from a specific module

### LINKED ISSUE #391: Add download button to download image from a specific module
In order to download the image, we need to click on it to do this. But if a button can be made to this, which is a good pattern to follow. 

UI for download button can be 

screenshot from 2018-10-06 02-32-26

Files need to be changed in order to add the button 
https://github.com/publiclab/image-sequencer/blob/2e7e042baab7e0bf43b821ea9e197cacf6efeac2/examples/lib/defaultHtmlStepUi.js#L40

### PR DESCRIPTION
**Pull request by Harshithpabbati**
Fixes #391 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/reviewers` and `@publiclab/is-reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Comment by Harshithpabbati** [comment]
@publiclab/is-reviewers

**Comment by Harshithpabbati** [comment]
> Have you changed the dist files directly? The changes shown are a bit weird

Wait for 10min i will check it.

File: dist/image-sequencer-ui.js [hunk 1]
```diff
@@ -396,6 +396,9 @@ function DefaultHtmlStepUi(_sequencer, options) {
     <button class="btn  insert-step" style="margin-left:10px;border-radius:6px;background-color:#fff;border:solid #bababa 1.1px;" >\
       <i class="fa fa-plus"></i> Add\
     </button>\
+    <button class="btn " style="margin-left:10px;border-radius:6px;background-color:#fff;border:solid #bababa 1.1px;" >\
```
Comment (jywarren) [inline dist/image-sequencer-ui.js]: Hmm, indeed this is in the dist files but not in the source files -- can you try adding there and then compiling? Thanks!!!

**Comment by Harshithpabbati** [comment]
Sure i will make the change 😁

**Comment by Harshithpabbati** [comment]
@jywarren do i directly implement it in boostrap panel https://github.com/publiclab/image-sequencer/issues/673

**Comment by Harshithpabbati** [comment]
#834 I sent the new pull request can you check it out @jywarren

**Pull request closed without merging**